Sources/tart/Commands/Run.swift

```diff
@@ -204,13 +218,18 @@ struct Run: AsyncParsableCommand {
       serialPorts.append(createSerialPortConfiguration(tty_read!, tty_write!))
     }
 
+    let useVNC = vnc || vncExperimental
+    let useBuiltInGraphics = (useVNC && graphics) || !noGraphics
+    let useAudio = (useBuiltInGraphics && !noAudio) || audio;
```

Comment:
`useVNC && graphics` seems a bit off. Can we just start passing `audio` down to `craftConfiguration` without changing these variables and logic.